docs/api-docs/api-requirements.md

```diff
@@ -0,0 +1,265 @@
+# API Requirements
+
+This document outlines some specific requirements for portal APIs that would be necessary
+for programmatic communication between BCDC, the specimen portal, and the sequencing portal to provide
+a central status dashboard for the consortium. This document is an extension of a general [requirements document](https://github.com/BICCN/BICCN-Infrastructure/blob/api-requirements/archive_communication/api-requirements.md)
+
+## Assumptions
+- This document and the examples make a few assumptions about the APIs:
+    - APIs communicate via JSON over HTTP
+    - APIs follow REST principles
```

Comment:
We should discuss more with consortium, but perhaps we could suggest interfaces use GraphQL or REST

Reply:
Sure, added an explicit note that graphql is a perfectly valid implementation of these requirements.